src/Arcus.Templates.WebApi/ExampleProviders/HealthReportExampleProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Swashbuckle.AspNetCore.Filters;
+
+namespace Arcus.Templates.WebApi.ExampleProviders
+{
+    public class HealthReportExampleProvider : IExamplesProvider<HealthReport>
+    {
+        public HealthReport GetExamples()
+        {
+            var entries = new Dictionary<string, HealthReportEntry>
+            {
+                ["api"] = new HealthReportEntry(HealthStatus.Healthy, "Api is healthy", TimeSpan.FromMilliseconds(120), null, null)
+            };
+            return new HealthReport(new ReadOnlyDictionary<string, HealthReportEntry>(new Dictionary<string, HealthReportEntry>()), TimeSpan.FromMilliseconds(201));
```

Comment:
Let's not inline these things. Something we do at Arcus is to make sure that we don't inline too much. Let's extract the read-only dictionary, maybe.

Reply:
Done.